tsdb/engine/tsm1/compact_test.go

```diff
@@ -4219,16 +4202,151 @@ func TestEnginePlanCompactions(t *testing.T) {
 				}
 
 				switch planType {
-				case tsm1.PT_Standard:
-					return common
-				case tsm1.PT_SmartOptimize:
+				case tsm1.PT_Standard, tsm1.PT_SmartOptimize:
 					return common
 				case tsm1.PT_NoOptimize:
 					return testLevelResults{}
 				}
 				return testLevelResults{}
 			},
 		},
+		{
+			name: "Mock another planned level inside scheduler",
+			files: []tsm1.ExtFileStat{
+				{
+					FileStat: tsm1.FileStat{
+						Path: "01-05.tsm",
+						Size: 256 * 1024 * 1024,
+					},
+					FirstBlockCount: tsdb.DefaultAggressiveMaxPointsPerBlock,
```

Comment:
Do we have a test where the file with the aggressive block count is not the first, but somewhere in the middle of file list?

Reply:
We should have the aggressive block count file first, last, and in the middle, in three different file groups or tests (Whichever is more convenient).